kubemarine/procedures/check_iaas.py

```diff
@@ -1372,6 +1372,50 @@ def check_ipip_tunnel(group: NodeGroup) -> Set[str]:
                               f"sudo rm -f {ipip_check} {ipip_check}.pid", warn=True)
 
 
+def fs_mount_options(cluster: KubernetesCluster) -> None:
+    with TestCase(cluster, '018', 'System', 'Filesystem mount options') as tc:
+
+        failed_nodes: Set[str] = set()
+        # Only Kubernetes nodes should be checked
+        group = cluster.make_group_from_roles(['control-plane', 'worker']).get_sudo_nodes()
+        # Get CRI root
+        if cluster.inventory['services']['cri']['containerRuntime'] == "containerd":
+            # Containerd root
+            if cluster.inventory['services']['cri']['containerdConfig'].get('root', ""):
+                cri_root = cluster.inventory['services']['cri']['containerdConfig']['root']
+            else:
+                cri_root = "/var/lib/containerd"
+        # Get rid of that part after KubeMarine stop supporting Kubernetes version lower v1.24
+        if cluster.inventory['services']['cri']['containerRuntime'] == "docker":
+            # Docker root
+            if cluster.inventory['services']['cri']['dockerConfig'].get('data-root', ""):
+                cri_root = cluster.inventory['services']['cri']['dockerConfig']['data-root']
+            else:
+                cri_root = "/var/lib/docker"
```

Comment:
```suggestion
        cri = cluster.inventory['services']['cri']
        cri_runtime = cri['containerRuntime']
        if cri_runtime == "containerd":
            # Containerd root
            cri_root = cri['containerdConfig']['root'] if cri['containerdConfig'].get('root') else "/var/lib/containerd"
        # Get rid of that part after KubeMarine stop supporting Kubernetes version lower v1.24
        elif cri_runtime == "docker":
            # Docker root
            cri_root = cri['dockerConfig']['data-root'] if cri['dockerConfig'].get('data-root') else "/var/lib/docker"
```

Reply:
Basically, KubeMarine could use various of CRI, not only `containerd` or `docker`.